### PR TITLE
Update locales

### PR DESCRIPTION
**Pull request by EKE00372**

### CODE REVIEW

File: locale/enUS.lua [hunk 1]
```diff
@@ -113,7 +115,10 @@ L['Opens this tutorial again.\nHope you liked it\n\n:)\n\n'] = true;
 L['Buy Items'] = true;
 L['AuctionFaster - Buy'] = true;
 L['Search'] = true;
-L['Filters'] = true;
```
Comment (pwilkowski) [inline locale/enUS.lua]: Why this one was removed? Was it duplicate?

File: locale/enUS.lua [hunk 2]
```diff
@@ -174,7 +179,9 @@ L['Use Custom Duration'] = true;
 L['12h'] = true;
 L['24h'] = true;
 L['48h'] = true;
-L['Auction Duration'] = true;
```
Comment (pwilkowski) [inline locale/enUS.lua]: Same question as above? Just duplicate?

File: locale/enUS.lua [hunk 3]
```diff
@@ -186,8 +193,6 @@ L['By default, AuctionFaster always undercuts price,\neven if you toggle "Rememb
 	'If you uncheck this option AuctionFaster\nwill never undercut items for you'] = true;
 
 -- sell tutorial
-L['Addon Tutorial'] = true;
-L['Addon settings'] = true;
```
Comment (pwilkowski) [inline locale/enUS.lua]: Again duplicates?

**Comment by pwilkowski** [comment]
Just to make sure, did you check any bugs after changing code? Possibly with bugsack + buggrabber?
For example if some entry is missing from translations, there will be error there.

File: locale/enUS.lua (hunk 1 above)
Comment (EKE00372) [reply locale/enUS.lua]: L['Filters'] repeat at line 70

File: locale/enUS.lua (hunk 2 above)
Comment (EKE00372) [reply locale/enUS.lua]: L['Auction Duration'] at line 23

File: locale/enUS.lua (hunk 3 above)
Comment (EKE00372) [reply locale/enUS.lua]: repeat at line 85 and 86

**Comment by EKE00372** [comment]
aww, I've test on retail zhTW clinet, locales work fine, but there's an error i saw and not sure what happened, it's my knowledge blind zone.

> integer overflow attempting to store inf

it happened when i start search auctions by click sell-tab's bag item list. first 1-5 try get this error, should try search another things then it will be ok.

**Comment by EKE00372** [comment]
have some question (caused by my poor english)

L['You still have %d of %s Do you wish to sell rest?'] is this mean ask for sell remaining items in bag?

**Comment by pwilkowski** [comment]
Remaining items in bag that you just tred to sell
for example you have 8 of something, your stack size is 5 and stack count = 0 = unlimited
this will ask if you wish to sell remaining 3

**Comment by pwilkowski** [comment]
Is that complete or you still need to do some changes?

**Comment by EKE00372** [comment]
I think those enough for use. will add other later.

But still trying that error, it wont happened on enUS client.

Maybe should mak a video to show it......I don't know how to describe that.

**Comment by pwilkowski** [comment]
Also if something like Qty and Lvl is invisible after translation, try stretching that column, its in

```
		{
			name         = L['Lvl'],
			width        = 35, -- here
			align        = 'LEFT',
			index        = 'level',
			format       = 'number',
		},
```

**Comment by pwilkowski** [comment]
yes video is the best. Although it may be just a problem with client?

**Comment by EKE00372** [comment]
https://youtu.be/gbxYRbZBIkQ
I try switch client locale between enUS/zhCN, no error. It happeneds only on zhTW clinet and some specific item, finally, I reset AuctionFaster SavedVariables, and all thins be fine.
https://youtu.be/IJMsiFOWtN8
-__- possible there were some magical issue broke format, when update to newest version?
so no more issue now, just need keep observe.

**Comment by EKE00372** [comment]
> Also if something like Qty and Lvl is invisible after translation, try stretching that column, its in
> 
> ```
> 		{
> 			name         = L['Lvl'],
> 			width        = 35, -- here
> 			align        = 'LEFT',
> 			index        = 'level',
> 			format       = 'number',
> 		},
> ```

Width up to 40 is enough to show them. but i'm not sure it will affect other frames or not.

**Comment by pwilkowski** [comment]
If you set this column to 40, you will have to cut 5 pixels from some other column. Basically whole length of all columns must remain the same to fit.

As for the bug you showed me, put this:
```
print(self.selectedItem.count, maxStacks, remainingQty)
```

Above the line 279 in 
`tabs\sell\functions.lua`

one of this variables is "Infinite" so we should debug this and see which one. I pushed attempted fix, please check if it works.

**Comment by EKE00372** [comment]
Reproduced this issue before update, it print 'inf' with error.
After update, no more error and print numbers normally, I think this fix effect indeed.

**Pull request merged**